src/components/common/Dropdown/Dropdown.tsx

```diff
@@ -1,10 +1,10 @@
 import { FC, ChangeEvent} from 'react';
-import { Units } from "../../types/types";
+import { Units } from "../../../types/units";
 import css from './styles.module.css'
 
 interface DropdownProps {
   value: Units;
-  units: {value: Units, label: string}[];
+  units: {value: Units, label: string, mark: string}[];
```

Comment:
компоненты в папке common должны мыть максимально абстрактными, т.е. применимы в  любой ситуации (контексте). Для этого нужно их пропсы тоже называть максимально абстрактно. units - это про погоду, options - просто опции дропдауна
```suggestion
  options: {value: Units, label: string, mark: string}[];
```